src/core/thread/csl_tx_scheduler.hpp

```diff
@@ -101,14 +101,59 @@ class CslTxScheduler : public InstanceLocator, private NonCopyable
         void     SetLastRxTimestamp(uint64_t aLastRxTimestamp) { mLastRxTimestamp = aLastRxTimestamp; }
 
     private:
-        uint8_t   mCslTxAttempts : 7;   ///< Number of CSL triggered tx attempts.
-        bool      mCslSynchronized : 1; ///< Indicates whether or not the child is CSL synchronized.
-        uint8_t   mCslChannel;          ///< The channel the device will listen on.
-        uint32_t  mCslTimeout;          ///< The sync timeout, in seconds.
-        uint16_t  mCslPeriod;           ///< CSL sampled listening period in units of 10 symbols (160 microseconds).
-        uint16_t  mCslPhase;            ///< The time when the next CSL sample will start.
-        TimeMilli mCslLastHeard;        ///< Time when last frame containing CSL IE was heard.
-        uint64_t  mLastRxTimestamp;     ///< Time when last frame containing CSL IE was received, in microseconds.
+        uint8_t  mCslTxAttempts : 7;   ///< Number of CSL triggered tx attempts.
+        bool     mCslSynchronized : 1; ///< Indicates whether or not the child is CSL synchronized.
+        uint8_t  mCslChannel;          ///< The channel the device will listen on.
+        uint32_t mCslTimeout;          ///< The sync timeout, in seconds.
+        uint16_t mCslPeriod; ///< CSL sampled listening period between consecutive channel samples in units of 10
+                             ///< symbols (160 microseconds).
+
+        /**
+         * The time in units of 10 symbols from the first symbol of the frame
+         * containing the CSL IE was transmitted until the next channel sample,
+         * see IEEE 802.15.4-2015, section 6.12.2.
+         *
+         * The Thread standard further defines the CSL phase (see Thread 1.3.1,
+         * section 3.2.6.3.4, also conforming to IEEE 802.15.4-2020, section
+         * 6.12.2.1):
+         *  * The "first symbol" from the definition SHALL be interpreted as the
+         *    first symbol of the MAC Header.
+         *  * "until the next channel sample":
+         *     * The CSL Receiver SHALL be ready to receive when the preamble
+         *       time T_pa as specified below is reached.
+         *     * The CSL Receiver SHOULD be ready to receive earlier than T_pa
+         *       and SHOULD stay ready to receive until after the time specified
+         *       in CSL Phase, according to the implementation and accuracy
+         *       expectations.
+         *     * The CSL Transmitter SHALL start transmitting the first symbol
+         *       of the preamble of the frame to transmit at the preamble time
+         *       T_pa = (CSL-Phase-Time – 192 μs) (that is, CCA must be
```

Comment:
@abtink : any policy for special characters in source? I recall that usually "us" or "microseconds" is used which avoids the mu symbol.

Reply:
Fixed.

Reply:
I don't believe we have any specific guideline or policy for this, but as @EskoDijk  mentioned, I think we have not used them in the past. It might be helpful to add a guideline on this later. I personally do like them, as they make the text look nicer, but I am not sure if they may cause any issues on different editors and/or older toolchains.

Reply:
@EskoDijk @abtink I did a search on OT source code and the mu symbol appears in a few places, but the us/microsecond notation is much more frequent (and I changed my docs accordingly). I think the symbol was introduced because I shamelessly copied it from the standard. But no matter what your decision, I'll adapt. No strong feelings re one or the other on my side.

Reply:
Thanks for checking it. Good idea to go with the more commonly used model (use usec/microseconds).